spec/features/two_factor_authentication/change_factor_spec.rb

```diff
@@ -49,15 +49,6 @@
         end
       end
     end
-
-    scenario 'deleting account' do
```

Comment:
cc @jmhooper this spec checked that we were prompted for 2fa before deleting account, basically testing that the controller inherited from the Reauth controller. I figured that by prompting for the password we'd be OK without, but I think that controller also checks another factor. Do you think it's still OK to remove this?

Reply:
Yep, I think this is obsolete now